bin/create-k8s-config-maps-and-secrets

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 HARMONY_VERSION=$(grep version ./package.json \
-  | head -1 \
-  | awk -F: '{ print $2 }' \
-  | sed 's/[",]//g' \
-  | tr -d '[[:space:]]')
+    | head -1 \
+    | awk -F: '{ print $2 }' \
+    | sed 's/[",]//g' \
+| tr -d '[[:space:]]')
```

Comment:
Looks like you removed leading whitespace that should probably remain

Reply:
my 'format on save' VS Code plugin seems to disagree. It wants to format it this way for some reason.